assets/src/application/ui/display/EntityIDs/index.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+import React, { useMemo } from 'react';
+import shortenGuid from '@appServices/utilities/text/shortenGuid';
+import { EntityId, EntityDbId } from '@edtrServices/apollo/types';
+
+import './style.css';
+
+export enum EntityIdAlignment {
+	LEFT = 'left',
+	RIGHT = 'right',
+}
+
+export interface EntityIdProps {
+	align?: EntityIdAlignment;
+	dbid: EntityDbId;
+	guid: EntityId;
+}
+
+const EntityIDs: React.FC<EntityIdProps> = ({ dbid, guid, align = 'left' }) => {
+	const htmlClass = classNames('ee-entity-ids', 'ee-focus-priority-9', {
```

Comment:
`ee-focus-priority-9` is interesting

Reply:
was thinking if we need to have these kind of classnames as enums, to have better control;
but this might be too much of typing; just thoughts out loud